drivers/serial/uart_npcx.c

```diff
@@ -367,6 +386,10 @@ static int uart_npcx_init(const struct device *dev)
 #endif
 
 #if defined(CONFIG_PM)
+	/* Initialize a miwu device input and its callback function */
+	npcx_miwu_init_dev_callback(&data->uart_rx_cb, &config->uart_rx_wui,
+						uart_npcx_rx_wk_isr, dev);
```

Comment:
line is over-indented (should align to left-paren of the function call)

Reply:
Done.